libgcc/config.host

```diff
@@ -1043,6 +1046,21 @@ powerpc-*-eabi*)
 	tmake_file="${tmake_file} rs6000/t-ppccomm rs6000/t-savresfgpr rs6000/t-crtstuff t-crtstuff-pic t-fdpbit"
 	extra_parts="$extra_parts crtbegin.o crtend.o crtbeginS.o crtendS.o crtbeginT.o ecrti.o ecrtn.o ncrti.o ncrtn.o"
 	;;
+powerpc*-*-hipperos*)
+    tmake_file="${tmake_file} rs6000/t-ppccomm rs6000/t-savresfgpr rs6000/t-crtstuff rs6000/t-linux t-dfprules rs6000/t-ppc64-fp t-slibgcc-libgcc"
```

Comment:
Why Linux? Why libgcc?

Reply:
Copy/pasted from the Linux implementation. Just needed to make it work, did not investigate further.

Reply:
`I would say that we should remove rs6000/t-linux